test/io/db_config.sql

```diff
@@ -35,6 +35,7 @@ ALTER ROLE db_config_authenticator SET pgrst.server_port = 'ignored';
 ALTER ROLE db_config_authenticator SET pgrst.server_unix_socket = 'ignored';
 ALTER ROLE db_config_authenticator SET pgrst.server_unix_socket_mode = 'ignored';
 ALTER ROLE db_config_authenticator SET pgrst.admin_server_port = 'ignored';
+ALTER ROLE db_config_authenticator SET pgrst.disable_recovery = 'true';
```

Comment:
For the [in-db configs](https://postgrest.org/en/stable/references/configuration.html#in-database-configuration) to work, we need to add them here: 

https://github.com/PostgREST/postgrest/blob/c820efb64a2839c5cb7a6a82b9145c3d7065474c/src/PostgREST/Config/Database.hs#L42-L66

That being said, I don't think we need to have an in-db config for this. It's a bit like changing the [db-uri](https://postgrest.org/en/stable/references/configuration.html#db-uri) (doesn't have an in-db config), which has the potential to kill the process.